engine/flink/schemed-kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/schemedkafka/schema/TestSchema.scala

```diff
@@ -1,13 +1,16 @@
 package pl.touk.nussknacker.engine.schemedkafka.schema
 
+import io.confluent.kafka.schemaregistry.avro.AvroSchema
 import org.apache.avro.Schema
 import org.apache.avro.generic.{GenericData, GenericRecord}
 import pl.touk.nussknacker.engine.api.validation.ValidationMode
 import pl.touk.nussknacker.engine.schemedkafka.AvroUtils
 import pl.touk.nussknacker.engine.schemedkafka.encode.ToAvroSchemaBasedEncoder
+import pl.touk.nussknacker.engine.schemedkafka.schemaregistry.confluent.ConfluentUtils
 
 trait TestSchema {
-  lazy val schema: Schema = AvroUtils.parseSchema(stringSchema)
+  lazy val schema: Schema              = AvroUtils.parseSchema(stringSchema)
+  lazy val confluentSchema: AvroSchema = ConfluentUtils.convertToAvroSchema(schema)
```

Comment:
Having two schemas next to each other is confusing. I thought about an approach, that we keep in samples this most commonly know type for schemas, which is easiest to build using DSL and to parse - `org.apache.avro.Schema`, and we pass it to `MockConfluentSchemaRegistryClientBuilder.register` where we convert it to `ParsedSchema` just for `MockSchemaRegistryClient.register` purpose.